modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx

```diff
@@ -121,7 +121,6 @@ const OrderableTableRow = ({
 			className={classNames('orderable-table-row', {
 				dragging: isDragging,
 			})}
-			key={index}
```

Comment:
This line was causing the bug that drag & drop was broken after first use. Indexes on React elements got messed up, and some rows stopped existing in virtual DOM.

Reply:
Good catch, this is an odd place for a "key" anyways since it isn't a direct dependent of a loop. Maybe it was due to a refactor or something. Or more likely, I was the one who added it 🤷